backend/src/common/services/azure/azure-ai.service.ts

```diff
@@ -0,0 +1,176 @@
+import fs from 'node:fs/promises';
+
+import * as SpeechSDK from 'microsoft-cognitiveservices-speech-sdk';
+
+import { type BaseConfig } from '~/common/config/base-config.package.js';
+import { HttpCode, HttpError, HTTPMethod } from '~/common/http/http.js';
+
+import { fileService } from '../services.js';
+import { type GetAvatarVoicesResponseDto } from './types/types.js';
+
+interface batchSynthesesAvatar {
+    avatarName: string;
+    avatarStyle: string;
+    textSSML: string;
+    voice: string;
+}
```

Comment:
Use types, not interfaces. Also, types and interfaces should start from upper-case

Reply:
Using type now, also checked upper-case starting